src/main/java/com/squareup/javawriter/ParameterWriter.java

```diff
@@ -18,15 +18,24 @@
 import java.io.IOException;
 
 public final class ParameterWriter extends VariableWriter {
+  private boolean isVararg;
+
   ParameterWriter(TypeName type, String name) {
     super(type, name);
   }
 
+  public void setIsVararg(boolean isVararg) {
```

Comment:
yea I spent a good five minutes trying to figure out a better nomenclature. I opted not to use setVarargs because it sounded like I was adding varargs to something.